addon/services/server-variables.js

```diff
@@ -1,18 +1,15 @@
-import Ember from 'ember';
-
-const {
-  isBlank,
-  String: { dasherize },
-  Service,
-} = Ember;
+import $ from 'jquery';
+import { isBlank } from '@ember/utils';
+import { dasherize } from '@ember/string';
+import Service from '@ember/service';
 
 export default Service.extend({
   unknownProperty: function(serverVar) {
     var ENV = this.get('env');
     var prefix = ENV.serverVariables.tagPrefix || ENV.modulePrefix;
     var dasherizedVar = dasherize(serverVar);
 
-    var content = Ember.$(`head meta[name=${prefix}-${dasherizedVar}]`).attr('content');
+    var content = $(`head meta[name=${prefix}-${dasherizedVar}]`).attr('content');
```

Comment:
Longer-term we can probably use native DOM here instead of jQuery, but that's probably out of scope for this PR.